src/ol_concourse/pipelines/open_edx/edx_platform_v2/grouped_docker_packer_pulumi_pipeline.py

```diff
@@ -0,0 +1,305 @@
+                # TODO MD 20230512 Fix to main branch once testing completed
+                branch="md/edxapp_docker_migration",
+                paths=[
+                    "src/bridge/settings/openedx/",
+                    "src/bilder/components/",
+                    "src/bilder/images/edxapp_v2/deploy.py",
+                    "src/bilder/images/edxapp_v2/group_data/",
+                    "src/bilder/images/edxapp_v2/files/",
+                    "src/bilder/images/edxapp_v2/templates/vector/",
+                    f"src/bilder/images/edxapp_v2/templates/edxapp/{deployment.deployment_name}/",  # noqa: E501
+                    "src/bilder/images/edxapp_v2/custom_install.pkr.hcl",
+                    f"src/bilder/images/edxapp_v2/packer_vars/{deployment.deployment_name}.pkrvars.hcl",  # noqa: E501
+                    f"src/bilder/images/edxapp_v2/packer_vars/{release_name}.pkrvars.hcl",  # noqa: E501
+                ],
+            )
+
+            edx_pulumi_code = git_repo(
+                name=Identifier(
+                    f"edxapp-ol-infrastructure-pulumi-{deployment.deployment_name}"
+                ),
+                uri="https://github.com/mitodl/ol-infrastructure",
+                # TODO MD 20230512 Fix to main branch once testing completed
+                branch="md/edxapp_docker_migration",
+                paths=[
+                    *PULUMI_WATCHED_PATHS,
+                    "src/ol_infrastructure/applications/edxapp/",
+                    "src/bridge/secrets/edxapp/",
+                    "src/bridge/settings/openedx/",
+                ],
+            )
+
+            edx_registry_image_resource = registry_image(
+                name=Identifier(
+                    f"edxapp-{release_name}-{deployment.deployment_name}-image"
+                ),
+                image_repository="mitodl/edxapp",
+                image_tag=f"{release_name}-{deployment.deployment_name}",
+                username="((dockerhub.username))",
+                password="((dockerhub.password))",  # noqa: S106
+            )
+
+            docker_build_inputs = (
+                [
+                    Input(name=edx_platform_git_resource.name)
+                    for edx_platform_git_resource in edx_platform_git_resources
+                ]
+                + [
+                    Input(name=theme_git_resource.name)
+                    for theme_git_resource in theme_git_resources
+                ]
+                + [Input(name=edx_docker_code.name)]
+            )
+
+            docker_build_job = Job(
+                name=f"build-{release_name}-{deployment.deployment_name}-edxapp-image",
+                build_log_retention={"builds": 10},
+                plan=[
+                    InParallelStep(
+                        in_parallel=theme_get_steps
+                        + edx_platform_get_steps
+                        + [GetStep(get=edx_docker_code.name, trigger=False)]
```

Comment:
This should also trigger a new build if there's platform code that needs to be executed